packages/transformers/js/src/JSTransformer.js

```diff
@@ -400,6 +400,7 @@ export default (new Transformer({
       supports_module_workers: supportsModuleWorkers,
       is_library: asset.env.isLibrary,
       is_esm_output: asset.env.outputFormat === 'esmodule',
+      trace_bailouts: options.logLevel === 'verbose',
```

Comment:
Should it just be a concern of the reporter to decide to show these? Is there significant overhead to always including them in diagnostics?

Reply:
Yeah I benchmarked and it does make a measurable difference in perf. Also uses a bit more memory. I imagine there are a lot of these bailouts in large apps.